plugins/modules/zos_job_query.py

```diff
@@ -208,18 +208,26 @@ def run_module():
         return result
 
     try:
-        validate_arguments(module.params)
-        jobs_raw = query_jobs(module.params)
-        jobs = parsing_jobs(jobs_raw)
+        name, id, owner = validate_arguments(module.params)
+        jobs_raw = query_jobs(name, id, owner)
+        if jobs_raw:
+            jobs = parsing_jobs(jobs_raw)
+        else:
+            jobs = None
+
     except Exception as e:
         module.fail_json(msg=e, **result)
     result["jobs"] = jobs
     module.exit_json(**result)
 
 
+# validate_arguments rturns a tuple, so we don't have to rebuild the job_name string
 def validate_arguments(params):
     job_name_in = params.get("job_name")
+    job_name_final = job_name_in
```

Comment:
job_name_final appears to not be used after this assignment

Reply:
forgot to remove it... was part of a test set.